callstack/react-native-paper
# fix(docs): Theming section

### Summary


This PR fix a theming documentation issue by adding some unrelated color properties. There are two missing color properties in the docs: 

- `notification`: specifies the default background color for badge components;
- `onSurface`: specifies the default background color for snackbar components;

It also remove an unused theme property: `onBackground`.

It will be useful for those who need to replace all default theme properties

### Test plan


- clone the project
- install dependencies using the command `yarn`
- Move to `/docs` directory and run `yarn start` command
- Check the new theme object specification on `http://localhost:3031/theming.html#theming`
- Compare it with the [current theme object specifications](https://callstack.github.io/react-native-paper/theming.html#theming)
- Move back to the root directory of the project and execute the tests by running `yarn test` command.

Before | After
----|----
![image](https://user-images.githubusercontent.com/26755665/111773451-f3ae0880-888c-11eb-80a4-ce4b4c6c0a44.png) | ![image](https://user-images.githubusercontent.com/26755665/111773515-09bbc900-888d-11eb-81a4-07f4d9193ff3.png)
